templates/rag-elasticsearch/rag_elasticsearch/chain.py

```diff
@@ -56,3 +57,11 @@ def _format_chat_history(chat_history: List[Tuple]) -> str:
 }
 
 chain = _inputs | _context | LLM_CONTEXT_PROMPT | llm | StrOutputParser()
+
+
+# Add typing for input
+class Question(BaseModel):
+    __root__: str
```

Comment:
I think this might be incorrect. Maybe something more similar to https://github.com/langchain-ai/langchain/pull/12682/files#diff-d26831cc3cbf5bf1acc93d4768150a54333b0c34d881d33e12f091dd50b18298R45-R49